Tests/Submit-Request.Tests.ps1

```diff
@@ -123,7 +121,9 @@ Describe -Name 'Private/Submit-Request' -Tag 'Private', 'Submit-Request' -Fixtur
     
     Context -Name 'Test empty result filtering' {
         Mock -CommandName 'Invoke-RubrikWebRequest' -Verifiable -MockWith {
-            '{"statuscode": 204}'
+            [PSCustomObject]@{
```

Comment:
I reverted all these back to reflect the fact that the WebResult objects are indeed objects coming from `Invoke-WebRequest`.